appendices/keywords.md

```diff
@@ -0,0 +1,60 @@
+# Keywords
+
+This listing explains the usage of every Pony keyword.
+
+|Keyword | Usage|
+| --- | --- |
+| actor | defines an actor
+| as | conversion of a value to another Type (can raise an error)
+| be | behavior, executed asynchronously
+| box | default reference capability – object is readable, but not writable
+| break | to step out of a loop statement
+| class | defines a class
+| compile_error | will provoke a compile error 
+| continue | loop statement
+| consume | move a value to a new variable, leaving the original variable empty
+| do | loop statement
+|    | after a with statement
+| else | conditional statement in if, for, while, repeat, try (as a catch block), match 
+| elseif | conditional statement, also used with ifdef
+| end | ending of if then, ifdef, while do, for in, repeat until, try, object, lambda,      |     | recover, match
+| error | raises an error
+| for | loop statement
+| fun | define a function, executed synchronously
+| if  | (1) conditional statement
+|     | (2) when specifying a guard in a pattern match
+| ifdef | when defining a build flag at compiling:  ponyc –D "foo"
+| in | loop statement
+| interface | used in structural subtyping
+| is | (1) used in nominal subtyping
+|    | (2) in type aliasing
+| iso | reference capability – read and write uniqueness
+| lambda | to make a closure
+| let | declaration of immutable variable
```

Comment:
the variable isn't immutable. what let does it say, you can't rebind this name. i'm not sure if this captures that distinction.